saleor/graphql/core/types/taxes.py

```diff
@@ -375,16 +375,16 @@ def resolve_discounts(root: Union[Checkout, Order], info: ResolveInfo):
         if isinstance(root, Checkout):
 
             def calculate_checkout_discounts(checkout_info):
-                is_shipping_voucher = (
-                    checkout_info.voucher.type == VoucherType.SHIPPING
-                    if checkout_info.voucher
-                    else False
-                )
                 checkout = checkout_info.checkout
                 discount_name = checkout.discount_name
                 return (
                     [{"name": discount_name, "amount": checkout.discount}]
-                    if checkout.discount and not is_shipping_voucher
+                    if checkout.discount
+                    and (
+                        checkout_info.voucher
+                        and checkout_info.voucher.type == VoucherType.ENTIRE_ORDER
```

Comment:
Can we also include this condition for static payload? I see that we already have the same condition for checkout-static-payload, but it looks like it is missing for orders: `saleor.webhook.payloads.generate_order_payload_for_tax_calculation`

Reply:
yep, will do

Reply:
@korycins done, please re-review when convenient.